apps/components-e2e/src/components/sunburst-chart/sunburst-chart.po.ts

```diff
@@ -18,18 +18,18 @@ import { Selector } from 'testcafe';
 
 export const body = Selector('body');
 export const chart = Selector('test-sunburst-chart');
-export const overlay = Selector('.dt-sunburst-chart-overlay-panel');
+export const overlay = Selector('.dt-overlay-container');
 
 export const centralLabel = Selector('.dt-sunburst-chart-selected-label');
 export const centralValue = Selector('.dt-sunburst-chart-selected-value');
 
 export const absoluteBtn = Selector('#chart-value-mode-absolute');
 export const percentBtn = Selector('#chart-value-mode-percent');
 
-export const segments = Selector('.dt-sunburst-chart-group');
+export const segments = Selector('[dt-sunburst-chart-segment]');
 export const sliceShephard = Selector(
-  '.dt-sunburst-chart-group:nth-of-type(5) .dt-sunburst-chart-slice',
+  '[dt-sunburst-chart-segment]:nth-of-type(5) .dt-sunburst-chart-slice',
```

Comment:
Could you please change the `nth-of-type` selector to the `nth` helper function, like we did for the stacked-series-chart.